src/job.ts

```diff
@@ -338,6 +343,13 @@ export class Job {
                 dockerCmd += `-e ${key}="${String(value).trim()}" `
             }
 
+            if (this.cache && this.cache.key && this.cache.paths) {
+                this.cache.paths.forEach((path) => {
+                    process.stdout.write(chalk`${jobNameStr} {magentaBright mounting cache} for path ${path}\n`);
+                    dockerCmd += `-v \${PWD}/.gitlab-ci-local/cache/${this.cache.key}/${path}:/builds/${path} `
```

Comment:
This will leave root owned files all over peoples project folders.
Using `/tmp/` would be a better place for these to live.

Reply:
By "all over" you mean in .gitlab-ci-local/, the working folder of our tool ?

On my ubuntu, files are not root owned :

```
> ls -lart .gitlab-ci-local/cache/maven/.cache
total 0
-rwxrwxr-- 1 bcouetil plugdev 0 avril  8 16:42 file1
drwxrwxr-- 1 bcouetil plugdev 0 avril  8 16:42 .
drwxrwxr-- 1 bcouetil plugdev 0 avril  8 17:05 ..
```
Can you check on your side ?

Reply:
Yes, I shall try it out...

Reply:
![image](https://user-images.githubusercontent.com/973602/114153369-889da200-991f-11eb-9467-778d24f8cd2a.png)
Yup, it will leave root owned files. @bcouetil

Reply:
OK, it's not ideal, but I guess we have no choice (I'll investigate a bit)

Reply:
Its either `/tmp/` or using `docker cp`
`docker cp` magicly makes sure permissions and user/group's are correct. @bcouetil

Reply:
I made a modification to have files removable by current user, please have a look @firecow 

`docker cp` is great, but Maven produces thousand of files, and would be significantly slower.

Reply:
I'm changing my mind.

there is a `docker cp`of the repo at the beginning of each job, and this copies the cache also.

I'm moving it to `/tmp/` as you said, to save time (I'm actually loosing 10s at the start of each job).

Reply:
If you are copying cache in and out using docker cp the tmp folder isn't needed.

Reply:
The `docker cp` I was mentioning is the one that populate `/build/`, not related to cache.

I still prefer not to copy cache while the `tmp` alternative is satisfying, because this would be 10s lost on each job in Maven context.

Reply:
Is it OK for you ? Then I guess we could merge :smile: